lib/src/component_declaration/component_base.dart

```diff
@@ -307,7 +370,7 @@ abstract class UiStatefulComponent<TProps extends UiProps, TState extends UiStat
 
   /// A typed state object corresponding to the current untyped state Map ([unwrappedState]).
   ///
-  /// Created using [typedStateFactory] and cached for each Map instance.
+  /// Created using [typedStateFactory] and cached for each Map instance within [_typedStateCache].
```

Comment:
#nit `within [_typedStateCache]` is an implementation detail not relevant to consumers of this API (and is also evident by looking at the code).